pom.xml

```diff
@@ -151,6 +151,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <slf4j.version>1.7.2</slf4j.version>
     <jackson.version>2.10.1</jackson.version>
+    <ozone.version>0.5.0-beta</ozone.version>
```

Comment:
Any reason specifically to use the beta? I would prefer to have official releases.

Reply:
Because for ozone, ozone will not have release version recently, and the beta version is the most stable version now, reference **https://hadoop.apache.org/ozone/downloads/**